eng/Directory.Build.Data.props

```diff
@@ -61,6 +61,8 @@
 
     <RequiredTargetFrameworks>net452;netstandard1.4;net461;netstandard2.0</RequiredTargetFrameworks>
     <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">netstandard2.0</RequiredTargetFrameworks>
+    
+    <TargetFrameworkMonikerAssemblyAttributesPath>$(IntermediateOutputPath)$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)</TargetFrameworkMonikerAssemblyAttributesPath>
```

Comment:
Is TargetFrameworkMoniker set at this point?

Reply:
Looking it the log it is set and used several times before this is called.

Reply:
We should simply double check to make sure it is set as expected for projects that have more then one target framework. Generally the TargetFrameworkMoniker is set from the TargetFramework so I just want to ensure it is set correctly at this point.